packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -1042,6 +1090,77 @@ export default class TabAdvanced extends Component {
                     </td>
                   </tr>
                 )}
+                {vm.boot.firmware === 'uefi' && (
+                  <tr>
+                    <th>{_('secureBoot')}</th>
+                    <td>
+                      <Toggle value={vm.secureBoot} onChange={value => editVm(vm, { secureBoot: value })} />
+                      <a
+                        className='text-muted'
+                        href='https://xcp-ng.org/docs/guides.html#guest-uefi-secure-boot'
+                        rel='noreferrer'
+                        style={{ display: 'block' }}
+                        target='_blank'
+                      >
+                        <Icon icon='info' /> {_('secureBootLinkToDocumentationMessage')}
+                      </a>
+                    </td>
+                  </tr>
+                )}
+                {vm.boot.firmware === 'uefi' && semver.satisfies(host?.version, '>=8.3.0') && (
```

Comment:
At the moment the warning "This feature is only available for UEFI VMs" is always displayed, this should be fixed when this PR will be merged, thanks to the fix in  #7767